misk/src/main/kotlin/misk/web/resources/StaticResourceEntry.kt

```diff
@@ -0,0 +1,23 @@
+package misk.web.resources
+
+/**
+ * This data class is used with Guice multibindings. Register instances by calling `multibind()`
+ * in a `KAbstractModule`:
+ *
+ * ```
+ * multibind<StaticResourceEntry>().toInstance(StaticResourceEntry(...))
+ * ```
+ */
+data class StaticResourceEntry(
+  override val url_path_prefix: String = "/",
+  private val resourcePath: String
+) : ResourceEntryCommon.Entry {
+  init {
+    ResourceEntryCommon.requireValidUrlPathPrefix(url_path_prefix)
+  }
+
+  fun resourcePath(urlPath: String): String {
+    val normalizedResourcePath = if (resourcePath.endsWith("/")) resourcePath.dropLast(1) else resourcePath
```

Comment:
BTW, this string could still end with `/` because the URL could be like `http://example.com/foo////` !